jobbergate-agent/jobbergate_agent/jobbergate/submit.py

```diff
@@ -72,26 +94,40 @@ async def submit_job_script(
 
         submit_dir = pending_job_submission.execution_directory or SETTINGS.DEFAULT_SLURM_WORK_DIR
 
-        job_script = None
-
-        for metadata in pending_job_submission.job_script.files:
-            local_script_path = submit_dir / metadata.filename
-            local_script_path.parent.mkdir(parents=True, exist_ok=True)
+        """
```

Comment:
This comment block may be an indication that the file's logic could be encapsulated in a function, what do you think?

Reply:
Good work on that